hmt_escrow/storage.py

```diff
@@ -163,7 +165,13 @@ def download(key: str, private_key: bytes, public: bool = False) -> Dict:
 
     """
     try:
-        content = download_from_storage(key=key, public=public)
+        url_pattern = "^https?:\\/\\/(?:www\\.)?[-a-zA-Z0-9@:%._\\+~#=]{1,256}\\.[a-zA-Z0-9()]{1,6}\\b(?:[-a-zA-Z0-9()@:%_\\+.~#?&\\/=]*)$"
+        is_url = re.match(url_pattern, key)
+        content = (
```

Comment:
why not just

if <condition>: 
  a
else:
 b

Reply:
this Yoda-like construction is cool, but a bit unclear what's the benefit?

Reply:
I have another opinion, may be it's from JS like ternary operator (? :), but I can change the statement if you wish.

Reply:
I wish not! Let it stay in.